File: WOT-iOS/WOTData/WOTData/CoreData/model/autogenerated/VehicleprofileArmor+CoreDataProperties.m
//
//  VehicleprofileArmor+CoreDataProperties.m
//  WOTData
//
//  Created by Pavel Yeshchyk on 1/17/20.
//  Copyright © 2020 Pavel Yeshchyk. All rights reserved.
//
//

#import "VehicleprofileArmor+CoreDataProperties.h"

@implementation VehicleprofileArmor (CoreDataProperties)

+ (NSFetchRequest<VehicleprofileArmor *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"VehicleprofileArmor"];
}

@dynamic front;
@dynamic rear;
@dynamic sides;
@dynamic vehicleprofileArmorListHull;
@dynamic vehicleprofileArmorListTurret;

@end

File: WOT-iOS/WOTData/WOTData/CoreData/model/autogenerated/VehicleprofileArmor+CoreDataProperties.swift
//
//  VehicleprofileArmor+CoreDataProperties.swift
//
//
//  Created by Pavel Yeshchyk on 4/23/20.
//
//

import Foundation
import CoreData

extension VehicleprofileArmor {
    @nonobjc public class func fetchRequest() -> NSFetchRequest<VehicleprofileArmor> {
        return NSFetchRequest<VehicleprofileArmor>(entityName: "VehicleprofileArmor")
    }

    @NSManaged public var front: NSDecimalNumber?
    @NSManaged public var rear: NSDecimalNumber?
    @NSManaged public var sides: NSDecimalNumber?
    @NSManaged public var vehicleprofileArmorListHull: VehicleprofileArmorList?
    @NSManaged public var vehicleprofileArmorListTurret: VehicleprofileArmorList?
}
